### DISCUSSION TITLE
Better way to view notes

### DISCUSSION THREAD

**Discussion by KraXen72** [iSoron/uhabits#1601] [Feature Requests]
Currently, to view past notes for days, you have to go to calendar statistics widget, click a gray edit button and view notes one by one.
  
i have a habit called **coding** and when i code, i usually write down what i worked on, in a sentence or two. it might be entirely possible this is out of scope and i should just write this somewhere else, but if not, **this is what i propose:**
#### a new view (bullet list) which has only the recent 5-7 days which have notes (full scrolling view upon click)
- the bullet list would look like this:
     - (day) (minutes/completed) the note for that day
     - (day) (minutes/completed) the note for the day...
  
 #### a tooltip upon holding on calendar widget squares, that shows the note
- the user is quickly able to see the time/completed + a note for that day. clicking on the tooltip opens full view.
  
 i am aware this might encourage users to put a lot of text into notes, so it might be sensible to limit notes to something like 180 or 280 characters (about a length of a tweet)

**Comment by mooror** [comment]
This is what I was looking for. I was going to make my own request if there wasn't one already. Another app that I have been using has the ability to view calendar months and for each month you can see a list of notes that have been added for that month

For me notes are a great way of tracking what I struggled with during the day for quiting habits I have. And I desire to be able to quickly read through a list of notes so I can see corralations. It also mist be useful to track and display at what time the note was added as well (though I could just write the time into the note too of course)

**Comment by herrbrixter** [comment]
I'd like a page to see each note added to a habit in chronological order. The past 10 days will do.

**Reply by herrbrixter** [reply]
The app HabitNow has this feature and it looks like this.

![Screenshot_20251206_231231~(1)](https://github.com/user-attachments/assets/05f2c52d-7502-4308-8500-32e6cfd9c86d)
